src/App.js

```diff
@@ -1,29 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+import './components/Calculator.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Calculator {
+  render() {
```

Comment:
- [ ] You also need to import `React, {Component}` and extend React `Component` instead of `Calculator` on line `4` as we are taking all the react component features to our own `App` class.